CRM/Contact/Form/Search.php

```diff
@@ -612,26 +612,27 @@ public function preProcess() {
           $this->_ssID = $ssId;
         }
       }
-
+      // set the uf group id if not already present
+      if (isset($this->_ufGroupID)) {
```

Comment:
The comment contradicts the code. I would recommend describing the use-case that this is fixing.